src/gebt_poc/mesh.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+
+#include <Kokkos_Core.hpp>
+
+namespace openturbine {
+  class Mesh {
+  public:
+    int GetNumberOfElements() {
+      return number_of_elements_;
+    }
+
+    int GetNumberOfNodes() {
+      return number_of_nodes_;
+    }
+
+    void SetNumberOfElements(int number_of_elements) { number_of_elements_ = number_of_elements; }
+    void SetNumberOfNodes(int number_of_nodes) { number_of_nodes_ = number_of_nodes; }
+
+    void InitializeElementNodeConnectivity(int number_of_elements, int nodes_per_element) {
```

Comment:
Maybe there should be a debug-mode assert here to make sure the number of elements and total nodes will end up consistent with the separately-settable values that the caller can give to `SetNumberOfElements` and `SetNumberOfNodes`.

Reply:
Checking element consistency should be trivial. Checking the consistency of the nodes will involve some reductions, since some nodes can exist in multiple elements.  Just checking if the maximum node index referenced in the connectivity is less than the number of nodes should be easy enough.

Reply:
I added a function to do some basic checks.  There probably are more checks to do to ensure consistency, so we can add more as we run into those edge cases.  Mesh construction will be happening under controlled conditions and the meshes will be pretty simple for this milestone, so I don't think we need much more at the moment.  By the time we get to more complex cases, I am assuming we'll need a more sophisticated data structure to handle them.